docs/vocascan-server/contribute.md

```diff
@@ -1,60 +1,62 @@
 # How to contribute
 
-1. Fork the [vocascan-server](https://github.com/vocascan/vocascan-server) repository
+## 1. Fork the&nbsp;[vocascan-server](https://github.com/vocascan/vocascan-server) repository
 
-   ![how to fork](../_media/images/vocascan-server-fork.png)
+![how to fork](../_media/images/vocascan-server-fork.png)
 
-2. Clone your fork
+## 2. Clone your fork
 
-   ```bash
-   git clone https://github.com/<username>/vocasan-server.git
-   ```
+```bash
+git clone https://github.com/<username>/vocasan-server.git
+```
 
-3. Checkout experimental
+## 3. Checkout experimental
 
-   ```bash
-   git checkout experimental
-   ```
+```bash
+git checkout experimental
+```
 
-4. Create your own branch
+## 4. Create your own branch
 
-   ```bash
-   git checkout -b my-awesome-feature
-   ```
+```bash
+git checkout -b my-awesome-feature
+```
 
-5. Install dependencies
+## 5. Install dependencies
 
-   ```bash
-   npm install --sqlite
-   ```
+```bash
+npm install --sqlite
+```
 
-6. Add config
+## 6. Add config
 
-   Copy the env config file template to configure your server.
+Copy the env config file template to configure your server.
 
-   ```bash
-   cp vocascan.config.example.js vocascan.config.js
-   ```
+```bash
+cp vocascan.config.example.js vocascan.config.js
+```
 
-   Fill out the config file. For a complete guide for each option see [configuration](vocascan-server/configuration)
+Fill out the config file. For a complete guide for each option see [configuration](vocascan-server/configuration)
```

Comment:
I would add in all 3 guides one sentence to this part that the server.jwt_secret and database.* options needs changes.